### DISCUSSION TITLE
개선사항

### DISCUSSION THREAD

**Discussion by Circlewee** [heading1/WYLSBingsu#193] [General]
# 개선사항
1. 쿠키 expire 설정
2. 깃헙 OAuth
3. ~~토핑에 닉네임도 같이 출력~~    **완료**
4. ~~webpack 배포 설정~~    **완료**
5. ~~자기 빙수에 글작성 막기~~    **완료**
6. 토핑 클릭 이벤트
7. 페이지네이션 개선 (skip 사용 X)
8. 코드 리뷰를 통한 코드 리팩토링
9. 신고하기 기능 만들어보기?
10. 비밀번호 찾기 프론트 구현
11. 사용 방법에 대해서 친절하게 알려줄수있는 기능
12. 랜덤 유저 방문
13. 비밀번호 찾기
14. 로그아웃

최신화: 22-07-06

**Comment by Circlewee** [comment]
추가할 내용은 comment로 남겨주세요

**Comment by jinux127** [comment]
토핑 클릭 이벤트 

예) 상세화면이 위에서 내려오는 이벤트 or 가운데서 점점 커지는 이벤트 등등

**Comment by kjklalala2** [comment]
- 페이지네이션 개선 (skip 사용 X)
- 코드 리뷰를 통한 코드 리팩토링
-  신고하기 기능 만들어보기?
-  비밀번호 찾기 프론트 구현

**Comment by sokojh** [comment]
사용 방법에 대해서 친절하게 알려줄수있는 기능이 있었으면 좋을 것 같아용 ㅎㅎ

**Comment by kjklalala2** [comment]
**카카오톡 연동 공유**

<위키 - 기획>
![image](https://user-images.githubusercontent.com/93240443/177022364-e20b0951-9337-469b-aba1-1368c15b352a.png)

- 자체 링크 공유 복사 기능을 이미 구현되어있어요

**Comment by Circlewee** [comment]
![image](https://user-images.githubusercontent.com/65718183/177033102-7bc49375-dd06-4aae-9626-2133ac12081b.png)
qa branch에 대한 protection을 추가

**Reply by sokojh** [reply]
오 이건 어떤 기능을 하는거죵?? 삭제하는것과 머지 체크 강제push 막는거 같은데

**Reply by Circlewee** [reply]
네 맞아요 default 브랜치에 대해 그런거 설정하는것 같아요
